samigo/samigo-app/src/webapp/jsf/evaluation/gradeStudentResult.jsp

```diff
@@ -134,7 +134,7 @@ function toPoint(id)
       <f:actionListener
         type="org.sakaiproject.tool.assessment.ui.listener.evaluation.QuestionScoreListener" />
     </h:commandLink>
-    <h:outputText value="</span><li role='menuitem'><span>" escape="false" rendered="#{totalScores.firstItem ne ''}" /> 
+    <h:outputText value="</span><li role='menuitem'><span>" escape="false" rendered="#{totalScores.firstItem ne '' && !totalScores.hasRandomDrawPart}" /> 
```

Comment:
I think you're going to have to split out the </span> here to just be on the #{totalScores.firstItem ne ''}

Also the closing </span><li> down below seems like should probably also be protected the same way so this is balanced?